generators/k8helmpipeline/README.md

```diff
@@ -0,0 +1,41 @@
+# K8helmpipeline
+
+The K8helmpipeline sub-generator creates a CI/CD pipeline in Visual Studio Teams Services (VSTS) and generates the template files needed to provision a basic Kubernetes web application using helm.
+
+## Installing
+
+Please refer to the [home](https://github.com/DarqueWarrior/generator-team) repository to install the Yeoman generator 
+
+
+
+
+## Prerequisites
+
+Along with an Azure Subscription and a VSTS Project, a few things have to be provisioned before running the generator: 
+
+1. A Kubernetes Cluster in either [Azure Container Services](https://docs.microsoft.com/en-us/azure/container-service/kubernetes/) (ACS) or [Azure Kubernetes Services](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS)
+
+2. An [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/) (ACR)
+
+   - The ACR's name

```

Comment:
ACR Name

Reply:
Also I think these might need examples, if I'm new to these terms, I'm not going to know what these are